### ISSUE TITLE
Website Stack

### ISSUE THREAD

**Issue by SaraVieira** [SaraVieira/webdev.wtf#2]
I was thinking of going with:

* Gatsby
* Netlify CMS
* Emotion
* Algolia

Anyone has ideas?

**Comment by hbuchel** [comment]
Just seconding, Gatsby + Emotion for styling have been my go-to lately.

**Comment by kayandra** [comment]
Or maybe the definitions could be json files, kinda like `ajax.json` that way, it could be easily available through GitHub.

We could even have multi lingual definitions in the json files

**Comment by stereobooster** [comment]
Gatsby, Netlify CMS, maybe Algolia, markdown files with frontmatter?

**Comment by cdvillard** [comment]
So far, Gatsby, Emotion, and Netlify CMS sound good. I second what @stereobooster suggested about Algolia or something like it for search. It was suggested in Website Ideas that there be a way to track undefined words that get popular. I think Algolia analytics can help there.

A question about Netlify CMS. I've seen it used for small teams, but given it's the direct tool for editorial workflow, does it offer user roles out of the box?

**Comment by SaraVieira** [comment]
@kayandra Netlufy CMS basically creates Markdown files so that anyone can edit them on GH too that's why I was thinking of using it. Every page can have an edit this on github that links to the markdown page

**Comment by SaraVieira** [comment]
If it's international I think it needs to be something like contenful :/

https://github.com/netlify/netlify-cms/issues/716

**Comment by SaraVieira** [comment]
@cdvillard I don't think it even has roles, you either have an account or you don't but since it creates Markdown files this CMS is really only needed in the start :)

**Comment by SaraVieira** [comment]
Added Algolia

**Comment by cdvillard** [comment]
Totally true. My inner journalism/content nerd was wondering.

**Comment by kayandra** [comment]
@saravieira I didn't know that's what they did. Then it sounds cool to me too

**Comment by SaraVieira** [comment]
YAAAY!

We will probably need to extend netlify cms but it's super extensible :)

**Comment by kayandra** [comment]
I'll wait for the weekend to read up on it. So far, it looks promising
